uint8 cmdflag
uint8 cmd
float32 J1
float32 J2
float32 J3
float32 J4
float32 J5
float32 J6
uint8 CO0
uint8 CO1
uint8 CO2
uint8 CO3
uint8 CO4
uint8 CO5
uint8 CO6
uint8 CO7
uint8 DO0
uint8 DO1
uint8 DO2
uint8 DO3
uint8 DO4
uint8 DO5
uint8 DO6
uint8 DO7
float32 tcp_x
float32 tcp_y
float32 tcp_z
float32 tcp_roll
float32 tcp_pitch
float32 tcp_yaw
